src/db/interviews.js

```diff
@@ -0,0 +1,51 @@
+module.exports = ((db) => {
+    const security = require('../security')
+
+    async function getAll(userId) {
+        let userInterviews = db.get('interviews')
+            .filter({ userId: userId })
+            .value()
+        if (!Array.isArray(userInterviews)) {
+            return []
+        }
```

Comment:
Check, whether this `if` clause is necessary in combination with lowdb's `filter`.